tox.ini

```diff
@@ -1,11 +1,13 @@
 [tox]
 envlist = py36,py37,py38,py39,py310,py311,pypy3
-minversion = 1.6
+minversion = 4.0
+isolated_build = True
 
 [testenv]
 usedevelop = True
 deps =
   sphinx
+  setuptools-scm
 extras =
   test
   twisted
```

Comment:
It's probably a bit much for this change but I recommend getting rid of `tox` and using Hatch's testing and development venv management.

Reply:
That makes sense, but I indeed think we should punt on it.